Explorer/Assets/Scripts/Global/SceneSharedContainer.cs

```diff
@@ -45,7 +45,7 @@ public class SceneSharedContainer
             {
                 SceneFactory = new SceneFactory(
                     ecsWorldFactory,
-                    new SceneRuntimeFactory(staticContainer.WebRequestsContainer.WebRequestController),
+                    new SceneRuntimeFactory(staticContainer.WebRequestsContainer.WebRequestController, realmData ?? new IRealmData.Fake()),
```

Comment:
I don't think we should pass `Fake` here as it's a production-ready class: better to pass the fake as an argument from the container that is "fake" on its own.
Moreover, `realmData` should not be `nullable` imo

Reply:
Yes, it's not be null but it is, so I isolated it this way

It might be better to have more refactor here